src/app/authentication/services/authentication.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { BaseService } from '@service/base.service';
+import { environment } from '@environment/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService extends BaseService {
```

Comment:
Could we use space line before this line?

Reply:
@junan I don't think we need to add a space between the class and its decorator 🤔